server/MoqGenerator/Interfaces/Lsp/IInterfaceGenericsBuilder.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using MoqGenerator.Model;
+
+#warning elevate interfaces up one level before? done with this branch, no need for everyting to be in Interfaces.Lsp -- or do a separate branch just for that
```

Comment:
just remove this warning, I don't want to rename files and have crappy git history